app/controllers/concerns/application_shared.rb

```diff
@@ -46,13 +46,13 @@ def set_taxonomy
               return false
             end
           end
-        elsif session.has_key?("#{taxonomy}_id") # session with taxonomy explicitly set to id or nil (any contexxt)
+        elsif session.has_key?("#{taxonomy}_id") # session with taxonomy explicitly set to id or nil (any context)
           if session["#{taxonomy}_id"].present?
             determined_taxonomy = find_session_taxonomy(taxonomy, user) # specific id
           else
             determined_taxonomy = nil # explicitly set any context
           end
-        else # user who didn't specify explicit context and does not have anything in session
+        elsif !user.admin? # user who didn't specify explicit context and does not have anything in session
```

Comment:
if we're changing this behavior back, would it make sense to do it for admins and users? I think we should be consistent at least for all users

I think keeping the else and setting `determined_taxonomy` to `nil` would we better